app/controllers/object_controller.rb

```diff
@@ -38,6 +38,10 @@ def druid
     params[:id]
   end
 
+  def version
+    params[:version] || 'v1'
```

Comment:
Is `v1` the right default? Shouldn't this be "whatever the latest head version is"?